codex-rs/protocol/src/custom_prompts.rs

```diff
@@ -0,0 +1,8 @@
+use serde::Deserialize;
+use serde::Serialize;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct CustomPrompt {
+    pub name: String,
+    pub content: String,
```

Comment:
Maybe send the `Path` back now, as well, so it's easier to add support for opening the custom prompt's file from the client going forward.